docs/changelog/plugin-push-message-v2.md

```
@@ -79,27 +79,30 @@ The new schema solves these by:
 | `delivery="passive"` | `ai_behavior="read"` |
 | `delivery="silent"` / `reply=False` | `visibility=["hud"], ai_behavior="blind"` |
 | `content="X"` | `parts=[{"type":"text","text":"X"}]` |
-| `binary_data=bytes, mime=...` | `parts=[{"type":"image","data":bytes,"mime":...}]` (or `audio`; `video` accepted in schema but main_server warn-drops it for now) |
-| `binary_url=URL` | `parts=[{"type":"image","url":URL}]` |
+| `binary_data=bytes, mime=...` | choose `type="image" | "audio" | "video"` from the MIME and use `parts=[{"type":...,"data":bytes,"mime":...}]` (`video` is accepted by the schema but currently warn-dropped by `main_server`) |
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)